docs/user-guides/backup-configuration.md

```diff
@@ -0,0 +1,248 @@
+# Backup configuration
+
+There are three types of options for the Vald cluster: backup, filtering, and core algorithms.
+
+This page describes how to enable the backup feature on your Vald cluster.
+
+## What is the backup
+
+Vald's backup function is to save the index data in each Vald Agent pod as a data file to the Persistent Volume or S3.
+When the Vald Agent pod is restarted for some reason, the index state is restored from the saved index data.
+
+## Backup methods
+
+You can choose one of three types of backup methods.
+
+- PV (recommended)
+- S3
+- PV + S3
+
+Please refer to the following tables and decide which method fit for your case.
+
+|             | PV                                                                                                                                                          | S3                                                                                                    | PV+S3                                                                                                                                            |
+| :---------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------- | :---------------------------------------------------------------------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------- |
+| usecase     | Want to use backup with low cost<BR>Would not like to use some external storage for backup<BR>Want to backup with highly compatible storage with Kubernetes | Want to use the same backup file with several Vald clusters<BR>Want to access the backup files easily | Want to use backup with PV basically and access the backup files easily<BR>Want to prevent backup file failure due to Kubernetes cluster failure |
+| merit :+1:  | Easy to use<BR>Highly compatible with Kubernetes<BR>Complete with internal network<BR>Safety backup if applying CoW                                         | Easy to access backup files<BR>It fan be shared and used by multiple clusters                         | The safest of these methods                                                                                                                      |
```

Comment:
```suggestion
| merit :+1:  | Easy to use<BR>Highly compatible with Kubernetes<BR>Complete with internal network<BR>Safety backup if applying CoW                                         | Easy to access backup files<BR>It can be shared and used by multiple clusters                         | The safest of these methods                                                                                                                      |
```